### PR TITLE
Fix for protractor exiting early

### PR DESCRIPTION
**Pull request by binary-bard**
- Also added a info function to logger for convenience

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,35 +3,38 @@ import { processResults } from './junit-xml'
...
 import Logger from './logger'
+import _ from 'lodash'
 
 export default function (options = {}, callback = function noop () {}) {
   let parsedOptions = parseOptions(options)
   let testAttempt = parsedOptions.testAttempt || 1
   let logger = new Logger(parsedOptions.color)
 
-  function rerunFailedTests () {
+  function rerunFailedTests (status, output) {
```
Comment (ashubham) [inline src/index.js]: What is `output` ? Write JsDOC or

File: dist/index.js [hunk 2]
```diff
@@ -28,31 +32,33 @@ exports['default'] = function () {
   var testAttempt = parsedOptions.testAttempt || 1;
   var logger = new _logger2['default'](parsedOptions.color);
 
-  function rerunFailedTests() {
+  function rerunFailedTests(status, output) {
```
Comment (ashubham) [inline dist/index.js]: `output` is not needed

File: src/index.js (hunk 1 above)
Comment (binary-bard) [reply src/index.js]: It is stdout and stdin from protractor's process.

File: dist/index.js (hunk 2 above)
Comment (binary-bard) [reply dist/index.js]: Ok, let me take that up separately after I verify we don't miss anything because of this.

**Pull request merged**